about.html

```diff
@@ -0,0 +1,285 @@
+            <div class="col-md-6 p-4">
+              <img class="imgstyle" src="assets/img/about2.png" alt="logo">
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="">
+      <section class="row first-nav pb-4 mt-md-3 pb-md-4">
+        <div class="row m-4">
+          <div class="col-12 text-center">
+            <h5 class="first-footer h5style1">
+              Partner
+            </h5>
+          </div>
+          <hr class="col-12">
+        </div>
+        <ul class="row justify-content-md-evenly align-items-md-center">
+          <li class="col-4 d-flex justify-content-center align-items-center col-md-2 d-md-flex justify-content-md-center align-items-md-center partner">mozilla</li>
+          <li class="col-4 d-flex justify-content-center align-items-center col-md-2 d-md-flex justify-content-md-center align-items-md-center partner">Google</li>
+          <li class="col-4 d-flex justify-content-center align-items-center col-md-2 d-md-flex justify-content-md-center align-items-md-center partner">Naver</li>
+          <li class="col-6 d-flex justify-content-center align-items-center col-md-2 d-md-flex justify-content-md-center align-items-md-center partner">daumkakao</li>
+          <li class="col-6 d-flex justify-content-center align-items-center col-md-2 d-md-flex justify-content-md-center align-items-md-center partner">airbnb</li>
+        </ul>
+      </section>
+      <section class="row mt-md-5 mb-md-5">
+        <div class="col-6 d-flex justify-content-center align-items-center col-md-5 d-md-flex justify-content-md-center align-items-md-center">
+          <a class="" href="index.html">
+            <img src="assets/img/favicon.png" class="img-fluid" alt="AKAM Logo">
+          </a>
+        </div>
+        <div class="col-6 col-md-7 hidden-mobile">
+          <div class="row mb-3">
+            <div class="col-md-12 d-md-flex justify-content-md-start align-items-md-center">
+              2015 Creative Commons Korea. Some rights reserved.
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12 d-md-flex justify-content-md-start align-items-md-center">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil inventore laboriosam dolores mollitia. Unde libero voluptatem voluptatum dolorem reiciendis. Atque neque corrupti cumque veritatis nostrum nam modi facilis nesciunt vitae.
+            </div>
+          </div>
+        </div>
+        <div class="col-6 shown-mobile">
+            <div class="row">
+              <div class="col-12 d-flex justify-content-start align-items-center">
+                2015 Creative Commons Korea.
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-12 d-flex justify-content-start align-items-center">
+                Some rights reserved.
+              </div>
+            </div>
+        </div>
+
+      </section>
+    </footer>
```

Comment:
Thank you for your kindly feedback @DeliceLydia. I believe that this part of the footer does not have to appear on mobile view at the bottom because on the design in the index part does not seem to appear that, but ok, I'm going to make it appear by removing the class that is in charge of that behavior. I really appreciate your comments. Your observations are now fixed on commit 8355fe0.